WordPress/Classes/ViewRelated/Post/PostCardStatusViewModel.swift

```diff
@@ -140,6 +140,12 @@ class PostCardStatusViewModel: NSObject {
         }
     }
 
+    func author(separatorDirection direction: UIUserInterfaceLayoutDirection) -> String {
```

Comment:
This separator logic feels overdone. Could we find a simpler way to make this look right? What about checking the mockups again and seeing if there's a better place to add the separator than in the ViewModel for the author label's text?

Reply:
Done. I've extracted the separator itself to the view controller.